mysite/ct/views.py

```diff
@@ -898,6 +900,7 @@ def unit_resources(request, course_id, unit_id):
 @login_required
 def wikipedia_concept(request, course_id, unit_id, source_id):
     'page for viewing or adding Wikipedia concept to this courselet'
```

Comment:
> We can adopt the standard """docstring""" format here.

It is an old code and not related to this PR.
Sure, we should refactor this but not in this PR - different PR type should be separated (feature, refactor old code etc.)